portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase

```diff
@@ -36,6 +36,55 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-94874."
+	@priority = "5"
+	@uitest
+	test CanEditSegmentIfHaveASelectInput {
+		property portal.acceptance = "true";
+
+		task ("Go to segments portlet") {
+			Segmentation.openSegmentsAdmin(siteURLKey = "test-site-name");
+		}
+
+		task ("Edit the segment with a select input") {
+			Segmentation.editSegment(entry = "EditSegment Test");
+
+			Segmentation.deleteProperties();
+
+			Segmentation.addProperties(propertyOption = "Regular Role");
+
+			Segmentation.fillProperty(selectCheckbox = "Administrator");
+
+			Segmentation.fillTitle(titleField = "EditSegmentIfHaveASelectInput Test");
+
+			Segmentation.viewMembers(memberName = "Test Test");
+
+			Button.clickSave();
+		}
+
+		task ("Assert segment edited") {
+			Segmentation.editSegment(entry = "EditSegmentIfHaveASelectInput Test");
+
+			Segmentation.viewSegment(
+				criterion = "select-input",
+				value = "Administrator");
+
+			Segmentation.fillProperty(selectCheckbox = "Power User");
+
+			Button.clickSave();
+
+			Segmentation.editSegment(entry = "EditSegmentIfHaveASelectInput Test");
+
+			AssertChecked.assertCheckedNotVisible(
+				checkboxName = "",
+				locator1 = "Checkbox#ANY_CHECKBOX_CHECKED");
```

Comment:
hi @Luizcarlosqueiroz !! :)

I added this new step to the tests to check the toggle, now it's more complete!! What do you think? 

Or would you prefer that I just leave this step in the entire test?

Reply:
Hey Rafa!
I think you could "merge" both steps I've created. For example, when user goes to edit segment, the toggle is enabled and checked (and user can edit it), but when the toggle is enabled and unchecked, the user can't edit the segment. Do you think this covers the scenario well?

Reply:
i got it!!